model/ResultsService.php

```diff
@@ -848,29 +848,45 @@ public function getVariableDataFromDeliveryResult($resultIdentifier, $wantedType
         return $this->extractTestVariables($this->getVariablesFromObjectResult($testCallIds), $wantedTypes);
     }
 
-    public function extractTestVariables($variableObjects, $wantedTypes)
+    public function extractTestVariables(array $variableObjects, array $wantedTypes, string $filter = self::VARIABLES_FILTER_ALL)
```

Comment:
Are we ready to deal with scalar type hints in tao? Did you run tests against this code?

Reply:
yes, tests are passing